### PR TITLE
Adds documentation for changing metadata store

### PR DESCRIPTION
**Pull request by saurav-c**
This PR adds documentation for changing the metadata store.

### CODE REVIEW

File: guides/changing-metadata-store.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Changing Metadata Store
+
+Aqueduct uses a storage layer for tracking workflow, operator, and artifact metadata. The default storage layer is the local filesystem. Aqueduct currently also supports using the following services for its metadata store:
```
Comment (vsreekanti) [inline guides/changing-metadata-store.md:3]: Let's add a little bit here about why the user might want to use a non-FS metadata store (and also when it's required).

File: guides/changing-metadata-store.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- Google Cloud Storage (GCS)
+
+The Aqueduct UI should be used to change the metadata store to AWS S3 or GCS. You
+do this by connecting a new integration of the respective service. As part of the
+integration connection dialog, you must select the box at the bottom that says
+`Use this integration for Aqueduct metadata storage.`.
+
+![](<../.gitbook/assets/change_metadata_store.png>)
+
+We will add support shortly for switching back to using the local filesyste.
```
Comment (vsreekanti) [inline guides/changing-metadata-store.md:14]: Typo: filesystem.

Comment (vsreekanti) [reply guides/changing-metadata-store.md:14]: Let's also clarify that it's not currently supported.

File: guides/changing-metadata-store.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Aqueduct uses a storage layer for tracking workflow, operator, and artifact metadata. The default storage layer is the local filesystem. Aqueduct currently also supports using the following services for its metadata store:
+- AWS S3
+- Google Cloud Storage (GCS)
+
+The Aqueduct UI should be used to change the metadata store to AWS S3 or GCS. You
+do this by connecting a new integration of the respective service. As part of the
+integration connection dialog, you must select the box at the bottom that says
+`Use this integration for Aqueduct metadata storage.`.
+
+![](<../.gitbook/assets/change_metadata_store.png>)
```
Comment (vsreekanti) [inline guides/changing-metadata-store.md:12]: For the image, can we take a screenshot of the full connection UI and then grey out everything but the box?

**Review by vsreekanti — CHANGES_REQUESTED** [review]
Looks good! Just a few places where we can use a little more detail.

File: guides/changing-metadata-store.md (hunk 3 above)
Comment (saurav-c) [reply guides/changing-metadata-store.md:18]: Is there a specific way you want me to do this? Just grey out the text?

**Review by vsreekanti — APPROVED** [review]

**Pull request merged**